core/trino-spi/src/main/java/io/trino/spi/security/PasswordAuthenticator.java

```diff
@@ -23,5 +24,16 @@ public interface PasswordAuthenticator
      * @return the authenticated entity
      * @throws AccessDeniedException if not allowed
      */
+    @Deprecated
```

Comment:
What plan do you have for removal of that method?

Why not to remove it now?

Same question to other authenticators.

Reply:
See my response here: https://github.com/trinodb/trino/pull/27387#discussion_r2653097267